### PR TITLE
Niloofar / Setup Coveralls 

### PR DESCRIPTION
**Pull request by niloofar-deriv**
Setup test coverage in our Github actions!

### CODE REVIEW

File: .github/workflows/pull-request.yml [hunk 1]
```diff
@@ -22,4 +22,7 @@ jobs:
       # run tests
       - name: Run tests
         run: npm run test
```
Comment (ako-deriv) [inline .github/workflows/pull-request.yml]:
```suggestion
        run: npm run test -- --coverage
```

File: package.json [hunk 2]
```diff
@@ -8,7 +8,7 @@
     "build": "tsc && vite build",
     "lint": "eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
     "preview": "vite preview",
-    "test": "vitest"
+    "test": "vitest run --coverage"
```
Comment (ako-deriv) [inline package.json]:
```suggestion
    "test": "vitest"
```

**Review by ako-deriv — CHANGES_REQUESTED** [review]

**Review by ako-deriv — DISMISSED** [review]

File: package.json (hunk 2 above)
Comment (matin-deriv) [reply package.json]: Can we add this "coverage": "vitest run --coverage"

Comment (matin-deriv) [reply package.json]: Can we add this "coverage": "vitest run --coverage"

**Review by matin-deriv — APPROVED** [review]

**Pull request merged**